### DISCUSSION TITLE
Compiled CSS output can't be loaded by browser because of MIME type

### DISCUSSION THREAD

**Discussion by germi** [tailwindlabs/tailwindcss#7399] [Help]
Used the easy installation instructions with Tailwind CLI: https://tailwindcss.com/docs/installation

The problem is that when I open the HTML document in the browser, the CSS won't load because of this:

```
Refused to apply style from 'http://127.0.0.1:5501/dist/output.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
```

Anybody knows how can I avoid this issue? I Googled around but couldn't find the solution.

Thanks!

**Comment by Chauban** [comment]
I have the same problem. Have u addressed it?

**Reply by mybearworld** [reply]
Try going to `http://127.0.0.1:5501/dist/output.css` (or whatever path is being output in the error message). This problem often appears if there is actually an error message (e.g. 404) there.
